calc.py

```
@@ -62,8 +62,16 @@ def opPrecedence(self,op1,op2):
 print("hello")
 
 
+print("hihi" , 10//0)
+
 prnt("hello")
 
+print("hi" == 123)
+
+
+oifbes("opdnnew")
```

Ruff (0.8.2)

Undefined name `oifbes`

(F821)